tests/csl-json/crossref-deep-review-csl/raw.json

```diff
@@ -0,0 +1,3814 @@
+        "authenticated-orcid": false,
+        "given": "David J.",
+        "family": "Harris",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0001-8931-9461",
+        "authenticated-orcid": false,
+        "given": "Dave",
+        "family": "DeCaprio",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0002-5796-7453",
+        "authenticated-orcid": false,
+        "given": "Yanjun",
+        "family": "Qi",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0003-3084-2287",
+        "authenticated-orcid": false,
+        "given": "Anshul",
+        "family": "Kundaje",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0001-9309-8331",
+        "authenticated-orcid": false,
+        "given": "Yifan",
+        "family": "Peng",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0001-6681-9754",
+        "authenticated-orcid": false,
+        "given": "Laura K.",
+        "family": "Wiley",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0001-8008-0546",
+        "authenticated-orcid": false,
+        "given": "Marwin H. S.",
+        "family": "Segler",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0002-1400-3398",
+        "authenticated-orcid": false,
+        "given": "Simina M.",
+        "family": "Boca",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0003-2191-0778",
+        "authenticated-orcid": false,
+        "given": "S. Joshua",
+        "family": "Swamidass",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0003-1349-4030",
+        "authenticated-orcid": false,
+        "given": "Austin",
+        "family": "Huang",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0002-5324-9833",
+        "authenticated-orcid": false,
+        "given": "Anthony",
+        "family": "Gitter",
+        "sequence": "additional",
+        "affiliation": []
+      },
+      {
+        "ORCID": "http://orcid.org/0000-0001-8713-9213",
+        "authenticated-orcid": false,
+        "given": "Casey S.",
+        "family": "Greene",
+        "sequence": "additional",
+        "affiliation": []
+      }
+    ],
+    "member": "175",
+    "published-online": {
+      "date-parts": [
+        [
+          2018,
+          4,
+          4
+        ]
+      ]
+    },
+    "reference": [
```

Comment:
Wow, I didn't realize all the references were included in the raw JSON.  This is a great case for pruning.

Reply:
Yes Crossref started including them. Note we already were filtering them from `references.json` as per:

https://github.com/greenelab/manubot/blob/fdafa3eb5969662598e5a602d14e161568a59a83/manubot/cite/citeproc.py#L50-L64

This PR generalizes the pruning, such that we don't need `citeproc_remove_keys`.